src/takeLastWhile.test.ts

```diff
@@ -0,0 +1,60 @@
+import { pipe } from './pipe';
+import { takeLastWhile } from './takeLastWhile';
+
+const DATA = [1, 2, 3, 4] as const;
```

Comment:
We don't need the DATA const; it makes tests easier to read when the data is inline.

Reply:
Done